bosh/releases/pre_render_scripts/asactors/operator/jobs/patch_operator_ctl.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+target="/var/vcap/all-releases/jobs-src/app-autoscaler/operator/templates/operator_ctl"
+sentinel="${target}.patch_sentinel"
+if [[ -f "${sentinel}" ]]; then
+  if sha256sum --check "${sentinel}" ; then
+    echo "Patch already applied. Skipping"
+    exit 0
+  fi
+  echo "Sentinel mismatch, re-patching"
+fi
+
+sed -i "s/pid_guard/#pid_guard/g" "${target}"
```

Comment:
For context, this is patching https://github.com/cloudfoundry/app-autoscaler-release/blob/3ae61ae696e0e067b3253a692a4be1566f7e2e0e/jobs/operator/templates/operator_ctl#L19

Where `pid_guard` is implemented in https://github.com/cloudfoundry/app-autoscaler-release/blob/master/src/common/utils.sh

Ultimately, then, our issue is that `/var/vcap/sys/run/` isn't expected to be reused between containers.  It looks like we just have a mount for [all of `/var/vcap/sys`](https://github.com/cloudfoundry-incubator/quarks-operator/blob/v4.5.6/pkg/bosh/manifest/job.go#L17) and nothing for `/var/vap/sys/run` in particular, so it gets reused.

Problem is that all volume mounts are attached to the pod (not the container), so they would get re-used across container restarts…  Not sure if there's a good way to go other than just patch the script.

Reply:
Yep, and the same in `scalingengine_ctl`.
Yep for `pid_guard` location.
Other way than patching the scripts - I did not see any either.